### PR TITLE
Expand mem.extra into columns

### PR DESCRIPTION
**Pull request by kk7ds**
- WIP: Expand mem.extra into columns
- Support enabling/disabling mem.extra expansion

### CODE REVIEW

**Comment by kk7ds** [comment]
This is not ready for primetime yet, but it kinda works for me:

image

Probably a lot of testing required to get comfortable with it, and a few other things I have to tweak. @KC9HI let me know what you think of this and ... please try to break it :)

cc: @sq5bpf

**Comment by KC9HI** [comment]
This appears to be working as expected. So far the only errors encountered are not related to #613. I will continue my testing.

**Comment by lunapiercook** [comment]
I received the SenHaiX 8600 yesterday and am currently working on adding it to the ga510.py driver for issue #10555, as the 8600 is (apparently) an earlier version of the 8800 series, which I added to the same driver earlier this year.
   
  
    
   
  
   The 8600 is advertised with the feature "Airband Receive" on the SenHaiX site, as well as on other dealer sites. This isn't a specific mode within the Menu of the radio. However, the radio can be manually programmed for frequencies down to 108 MHz. This covers the airbands, which are from 108 MHz to 137 MHz. Listening here in Maine with the 8600, I can clearly hear air traffic overhead on the approach and departure frequencies for Bangor, Auburn, and Portland, even though air traffic is AM. (The radio being FM means transmission isn't allowed on those frequencies.)
   
  
    
   
  
   Extrapolating this information, I then tested the Radioddity GS-5B at those frequencies. This radio is a rebranded SenHaiX 8800. I could also tune this radio down to 108 MHz, and listen to the same traffic. This means the 8800 series is also capable of Airband Receive.
   
  
    
   
  
   I then tested the GA-510, which is not capable of being tuned to airbands.
   
  
    
   
  
   Changing the SHX8800 subclass in the ga510.py driver allows Chirp to program the 8600/8800 series down to 108 MHz, without affecting Chirp programming frequency limits for the GA-510.
   
  
    
   
  
   On a similar note, my son helps manage autocross when events occur on a decommissioned runway at Toledo Express Airport. An Extra class ham, he uses a Baofang UV-5R to listen to tower communications in case the event needs to clear the area due to any situation. This is because he can manually tune the UV-5R down to 108 MHz to monitor tower comms. Chirp's driver for the UV-5R, however, has a low end of 200 MHz.
   
  
    
   
  
   The bottom line are these questions:
   
  
    
   
  
   1) Is it alright to add airband receive bands down to 108 MHz to radios not advertised as such, or should they be only be added to those with that feature "as-advertised", such as the 8600?
   
  
   2) For frequencies from 108 - 137 MHz, would it be prudent to add "AIRBAND" to the Name field in memory if that field is empty?
   
  
    
   
  
   Working on not enough coffee for this...
   
  
    
   
  
   Dave Liske

**Comment by kk7ds** [comment]
@lunapiercook Why are you bringing up this question in this otherwise unrelated PR? Let's please not do that and confuse this PR with your question/discussion. You need an issue on the chirp site for the new model anyway, so that would be a good place. Or the dev mailing list, or on your own PR proposing it.

That said, I agree with @KC9HI - unless it's impossible to hide the setting from the radios that don't support it, it should be hidden. As soon as you add a thing that doesn't have an effect, people will ask why it doesn't work. If it _does_ work, then it's fine.

And no, you can't mangle the name field for things like this. The name field is the alpha tag for a channel and is "owned" by the user, not the driver.

But please, if we need further discussion on this, let's do it in a place that is contextually-related to your change and not this PR (#613). Thanks!

**Comment by lunapiercook** [comment]
I have no idea how the Subject line became what it is. I specified issue #10555 in the Subject field when I sent it, and it's that way in my Sent folder. It was not at all what it is now. That was not my doing. I'll proceed... Thanks. 
-------- Original message --------From: Dan Smith ***@***.***> Date: 6/1/23  11:18 AM  (GMT-05:00) To: kk7ds/chirp ***@***.***> Cc: lunapiercook ***@***.***>, Mention ***@***.***> Subject: Re: [kk7ds/chirp] Expand mem.extra into columns (PR #613) 
@lunapiercook Why are you bringing up this question in this otherwise unrelated PR? Let's please not do that and confuse this PR with your question/discussion. You need an issue on the chirp site for the new model anyway, so that would be a good place. Or the dev mailing list, or on your own PR proposing it.
That said, I agree with @KC9HI - unless it's impossible to hide the setting from the radios that don't support it, it should be hidden. As soon as you add a thing that doesn't have an effect, people will ask why it doesn't work. If it does work, then it's fine.
And no, you can't mangle the name field for things like this. The name field is the alpha tag for a channel and is "owned" by the user, not the driver.
But please, if we need further discussion on this, let's do it in a place that is contextually-related to your change and not this PR (#613). Thanks!

**Comment by lunapiercook** [comment]
Every ham I've ever met is gracious... Except you. I'm done, as I'm obviously not welcome here. Take a hike, and fix this shit yourself. 
-------- Original message --------From: Dan Smith ***@***.***> Date: 6/1/23  11:24 AM  (GMT-05:00) To: kk7ds/chirp ***@***.***> Cc: lunapiercook ***@***.***>, Mention ***@***.***> Subject: Re: [kk7ds/chirp] Expand mem.extra into columns (PR #613) 
@lunapiercook You're replying to github email notifications as if they were from a mailing list, which means the discussion is tagged to this PR (#613) forever in the archive. Please, don't reply to github notifications with general discussion. It's not a mailing list and it means your replies will likely get attached to unrelated things, as they are in this case.
